WcaOnRails/app/webpacker/javascript/event_navigation/index.js

```diff
@@ -0,0 +1,31 @@
+import classnames from 'classnames';
+import { Popup, Menu } from 'semantic-ui-react';
+import React from 'react';
+import EventIcon from '../wca/EventIcon';
+import events from '../wca/events.js.erb';
+import './index.scss';
+
+const EventNavigation = ({ selected, eventIds, onSelect }) => (
+  <Menu text>
+    {eventIds.map((event, index) => (
```

Comment:
Nit: please name this `eventId` instead, as it may cause some confusion. For instance there's `eventIds[index]` below, whereas it could just be `eventId`.